exporter/awsemfexporter/config.go

```diff
@@ -64,6 +64,8 @@ type Config struct {
 	// TODO: we can support directing output to a file (in the future) while customer specifies a file path here.
 	OutputDestination string `mapstructure:"output_destination"`
 
+	CreateHighLevelObject bool `mapstructure:"create_high_level_object"`
```

Comment:
Let's add a comment. Also, I believe its only for EKS Fargate customers. Let's make it clear renaming the varibale. `EKSFargateMetadata` or `CreateEKSFargateKubernetesObject`?